src/utils/SpokeUtils.ts

```diff
@@ -177,3 +178,45 @@ export function relayFilledAmount(
 
   return spokePool.relayFills(hash, { blockTag });
 }
+
+export async function findFillBlock(
```

Comment:
todo: missing documentation for `fiindFillBlock()`.

Reply:
https://github.com/across-protocol/sdk-v2/pull/488/commits/0b4a2dd1b6dbe33160cc02a343297939a4463215